lib/prometheus_exporter/middleware.rb

```diff
@@ -58,6 +58,13 @@ def default_labels(env, result)
     if params
       action = params["action"]
       controller = params["controller"]
+    elsif env["rack.cors"] && env["rack.cors"].respond_to?(:preflight?) && env["rack.cors"].preflight?
```

Comment:
last minor one, no need to look up in the hash 3 times. might as well store in a local var then do the tests on that.

Except for that seems fine.

Reply:
Changed that too, thanks!